pkgs/shells/carapace/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "carapace";
-  version = "1.0.5";
+  version = "1.0.6";
 
   src = fetchFromGitHub {
-    owner = "rsteube";
+    owner = "${pname}-sh";
```

```suggestion
    owner = "carapace-sh";
```

we consider using pname an anti-pattern